app/src/main/java/com/example/una/adapters/HorizontalRecyclerViewAdapter.java

```diff
@@ -0,0 +1,62 @@
+package com.example.una.adapters;
+
+import android.content.Context;
+import android.support.annotation.NonNull;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.TextView;
+import android.widget.Toast;
+
+import com.example.una.R;
+import com.example.una.RecyclerViewModels.HorizontalModel;
+
+import java.util.ArrayList;
+
+public class HorizontalRecyclerViewAdapter extends RecyclerView.Adapter<HorizontalRecyclerViewAdapter.HorizontalRVViewHolder> {
+
+    private final ArrayList<HorizontalModel> arrayList;
+    Context context;
+    public HorizontalRecyclerViewAdapter(Context context, ArrayList<HorizontalModel> arrayList) {
+        this.context = context;
+        this.arrayList = arrayList;
+    }
+    @NonNull
+    @Override
+    public HorizontalRecyclerViewAdapter.HorizontalRVViewHolder onCreateViewHolder(@NonNull ViewGroup viewGroup, int i) {
```

Comment:
let's not use single letter variable names. They aren't descriptive at all. the `i` here represents view type so let's call it `viewType`.

Reply:
You can find this info in the android documentation https://developer.android.com/reference/android/support/v7/widget/RecyclerView.Adapter.html#createViewHolder(android.view.ViewGroup,%20int)